deepdish/io/hdf5io.py

```diff
@@ -270,7 +280,17 @@ def _load_pickled(level):
 
 def _load_level(handler, level):
     if isinstance(level, tables.Group):
-        dct = {}
+        if _sns and (level._v_title.startswith('sns:') or
+                     DEEPDISH_IO_ROOT_IS_SNS in level._v_attrs):
+            val = sns()
+            dct = val.__dict__
```

Comment:
`dct` refers to `val` in the SimpleNamespace block, but in the else block, `val` refers to `dct`. It makes the rest of the function agnostic about whether it's reading a SimpleNamespace or a dict since `val` is the return in either case. Seem good?